packages/element/src/renderElement.ts

```diff
@@ -659,6 +662,16 @@ const drawElementFromCanvas = (
     // revert afterwards we don't have account for it during drawing
     context.translate(-cx, -cy);
 
+    const isDarkMode = appState.theme === 'dark';
```

Comment:
By convention it should be `const isDarkTheme = this.state.theme === THEME.DARK;`